### ISSUE TITLE
Lookup does not find entries in Dictionary on Catalina (worked on previous OS)

### ISSUE THREAD

**Issue by bjinse** [brentsimmons/NetNewsWire#1126]
When I press ctrl-cmd-D or use right-click / lookup, the results window does not find any entries in Dictionary.
Looking up the same word in e.g. Safari does work and does show entries.

Config:
Just installed Catalina / Before Catalina this worked.
NNW: Version 5.0.2 (2615)
Main language on Mac OS: Dutch
Language NNW: English

I have several English dictionaries installed, among which Apple Dictionary and a Dutch-English dictionary (Prisma).
In Spotlight preferences, I have the Setting to show Spotlight-suggestions in Lookup OFF.
When I turn it on, NNW finds Spotlight suggestions, like info on Wikipedia, but not from dictionaries.
(When off, nothing is found).

**Comment by bjinse** [comment]
I thought I should fiddle with my primary language to see if this fixed it. I saw I had an unusual language (Greek...) as my second language, before English, but after Dutch.
It was in there to show off the language capabilities in Mac OS, not actually in use...
When I removed Greek from the settings, and restarted NNW, lookup worked again.
So my language order was: Dutch / Greek / English when it did not work.
I tried to reproduce by adding Greek again as second language. Nope, issue is not reproducible. Lookup works as a charm.
Thanks to a tip from John Gruber, now without the annoying Spotlight suggestions.
I need lookup to improve my English, so I am happy it works again.
But if others stumble into the same issue, I might as well have it documented here.
I guess this one can ultimately be closed as not reproducible.
